src/server/scripts/Events/pilgrims_bounty.cpp

```diff
@@ -61,10 +62,19 @@ class spell_pilgrims_bounty_buff_food : public SpellScriptLoader
                 _handled = true;
                 GetTarget()->CastSpell(GetTarget(), _triggeredSpellId, true);
             }
+            
+            void FoodBonus(AuraEffect const* /*aurEff*/, int32& amount, bool& canBeRecalculated)
+            {
+                amount = CalculatePct(GetCaster()->GetMaxPower(POWER_MANA), 20);
+                amount = CalculatePct(GetCaster()->GetMaxHealth(), 15);
```

Comment:
are you sure this code is doing what you want ? `amount` is assigned twice in a row, with the first value being discarded

Reply:
Without a doubt it was not. This PR causes startup errors on spell script validation (Register() part)

Reply:
Sorry, I didn't notice the startup error in the white font